dbtvault-dev/macros/internal/escape_column_name.sql

```diff
@@ -0,0 +1,71 @@
+{%- macro escape_column_name(columns=none) -%}
+
+{# Different platforms use different escape characters, the default below is for Snowflake which uses double quotes #}
+
+    {%- set escape_char_left  = var('escape_char_left',  '"') -%}
```

Comment:
What's the reason for having a different escape char for the left and right? Surely we should have the same for both no matter what?

Reply:
Older MSSQL uses [ ] to escape column names.